MAGECLOUD-3913: Deadlocks when consumers run from cron (#5141)
magento/devdocs: _includes/config/message-queue-consumers.md

--- | --- | --- | ---
`--max-messages=<value>` | The maximum number of messages to consume per invocation. If the number of queued messages is less than the specified max, the consumer polls for new messages until it has processed the max. If you don't specify `--max-messages`, the process runs continuously. | No | 0
`--batch-size=<value>` | The number of messages to consume per batch. If specified, messages in a queue are consumed in batches of `<value>` each. This option is applicable for the batch consumer only. If `--batch-size` is not defined, the batch consumer receives all available messages in a queue. | No | 0
`--pid-file-path=<value>` | This option is deprecated, use `--single-thread` instead. | No | 
`--area-code=<value>` | The area code preferred for consumer process. | No | global

> `--single-thread` | This option prevents running multiple copies of one consumer simultaneously. | No | 

Reviewer: ```suggestion
`--single-thread` | This option prevents multiple copies of a consumer from running simultaneously. | No | 
```

`<consumer_name>` | The consumer to start. | Yes | |

{: .bs-callout .bs-callout-warning }

> The option `--pid-file-path` was marked as deprecated in releases 2.2.10 and 2.3.3 and pid files won't be generated anymore.

Reviewer: ```suggestion
The `--pid-file-path` option is deprecated in releases 2.2.10 and 2.3.3. Magento will no longer generate pid files.
```
